contracts/test/psm/PegStabilityModuleNonCustodial.t.sol

```diff
@@ -0,0 +1,235 @@
+pragma solidity ^0.8.4;
+
+import {ERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import {MockPCVDepositV2} from "../../mock/MockPCVDepositV2.sol";
+import {IPCVDeposit} from "../../pcv/IPCVDeposit.sol";
+import {MockERC20} from "../../mock/MockERC20.sol";
+import {MockOracle} from "../../mock/MockOracle.sol";
+import {ICore} from "../../core/ICore.sol";
+import {Core} from "../../core/Core.sol";
+import {IFei, Fei} from "../../fei/Fei.sol";
+import {PegStabilityModuleNonCustodial} from "./../../peg/PegStabilityModuleNonCustodial.sol";
+import {PegStabilityModule} from "./../../peg/PegStabilityModule.sol";
+import {Vm} from "./../utils/Vm.sol";
+import {DSTest} from "./../utils/DSTest.sol";
+import {getCore, getAddresses, FeiTestAddresses} from "./../utils/Fixtures.sol";
+
+contract PegStabilityModuleNonCustodialTest is DSTest {
+
+    PegStabilityModuleNonCustodial private psm;
+    ICore private core;
+    IFei private fei;
+
+    modifier prank(address toPrank) {
```

Comment:
Wondering if we can make the tests cleaner than having to add this modifier to each test. Why not just set the `userAddress` to be this contract (`address(this)`) and then mint Fei to this contract. Then we can remove the modifier from each unit test

Reply:
good idea. accepted!